contract-testing/src/impls.rs

```diff
@@ -2058,6 +2060,12 @@ pub fn module_load_v1(module_path: impl AsRef<Path>) -> Result<WasmModule, Modul
     Ok(module)
 }
 
+pub fn module_load_contract() -> Result<WasmModule, ContractModuleLoadError> {
```

Comment:
**Suggestion:** I try not to care about naming things, but here I think the name should be more explicit of what it does compared to the `module_load_v1`.
Maybe `module_load_output`?

Reply:
I like your suggestion better.